internal/stream-share/command_consumer_redeem.go

```diff
@@ -13,21 +13,38 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
-type redeemOut struct {
-	Id                         string   `human:"ID" serialized:"id"`
-	ApiKey                     string   `human:"API Key" serialized:"api_key"`
-	ApiSecret                  string   `human:"API Secret" serialized:"api_secret"`
-	KafkaBootstrapUrl          string   `human:"Kafka Bootstrap URL" serialized:"kafka_bootstrap_url"`
-	SchemaRegistryApiKey       string   `human:"Schema Registry API Key" serialized:"schema_registry_api_key"`
-	SchemaRegistrySecret       string   `human:"Schema Registry Secret" serialized:"schema_registry_secret"`
-	SchemaRegistryUrl          string   `human:"Schema Registry URL" serialized:"schema_registry_url"`
-	Resources                  []string `human:"Resources" serialized:"resources"`
-	NetworkDnsDomain           string   `human:"Network DNS Domain" serialized:"network_dns_domain"`
-	NetworkZones               string   `human:"Network Zones" serialized:"network_zones"`
-	NetworkZonalSubdomains     []string `human:"Network Zonal Subdomains" serialized:"network_zonal_subdomains"`
-	NetworkKind                string   `human:"Network Kind" serialized:"network_kind"`
-	NetworkPrivateLinkDataType string   `human:"Network Private Link Data Type" serialized:"network_private_link_data_type"`
-	NetworkPrivateLinkData     string   `human:"Network Private Link Data" serialized:"network_private_link_data"`
+type redeemHumanOut struct {
+	Id                         string `human:"ID"`
+	ApiKey                     string `human:"API Key"`
+	ApiSecret                  string `human:"API Secret"`
+	KafkaBootstrapUrl          string `human:"Kafka Bootstrap URL"`
+	SchemaRegistryApiKey       string `human:"Schema Registry API Key"`
+	SchemaRegistrySecret       string `human:"Schema Registry Secret"`
+	SchemaRegistryUrl          string `human:"Schema Registry URL"`
+	Resources                  string `human:"Resources"`
+	NetworkDnsDomain           string `human:"Network DNS Domain"`
+	NetworkZones               string `human:"Network Zones"`
+	NetworkZonalSubdomains     string `human:"Network Zonal Subdomains"`
+	NetworkKind                string `human:"Network Kind"`
+	NetworkPrivateLinkDataType string `human:"Network Private Link Data Type"`
+	NetworkPrivateLinkData     string `human:"Network Private Link Data"`
+}
+
+type redeemSerializedOut struct {
+	Id                         string   `serialized:"id"`
+	ApiKey                     string   `serialized:"api_key"`
+	ApiSecret                  string   `serialized:"api_secret"`
+	KafkaBootstrapUrl          string   `serialized:"kafka_bootstrap_url"`
+	SchemaRegistryApiKey       string   `serialized:"schema_registry_api_key"`
+	SchemaRegistrySecret       string   `serialized:"schema_registry_secret"`
```

Comment:
api secret?

Reply:
Thoughts on renaming this field to "API secret"? @confluentinc/data-exchange